dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -519,6 +593,27 @@ await foreach (StreamingChatCompletionsUpdate update in response.ConfigureAwait(
                     continue;
                 }
 
+                try
```

Comment:
Still need test coverage for streaming. I am having issues getting the tests running, probably due to not mocking the streaming model responses properly. Working on it...

Reply:
@gitri-ms Here is unit test for streaming and function call scenario, maybe it will help:
https://github.com/microsoft/semantic-kernel/blob/6652ce61c20ef8081a5dd3cc211a53e933d53e15/dotnet/src/Connectors/Connectors.UnitTests/OpenAI/ChatCompletion/AzureOpenAIChatCompletionServiceTests.cs#L436